src/libraries/System.Xml.XDocument/ref/System.Xml.XDocument.cs

```diff
@@ -489,10 +489,10 @@ public partial class XStreamingElement
     {
         public XStreamingElement(System.Xml.Linq.XName name) { }
         public XStreamingElement(System.Xml.Linq.XName name, object content) { }
```

Comment:
Should this be `object? content`?

Reply:
good catch, seems it's already `object?` elsewhere